tests/network-ovn

```diff
@@ -215,7 +215,7 @@ ovn_basic_tests() {
     [ "${U3_IPV4_OLD}" = "${U3_IPV4}" ]
 
     echo "===> Testing project restrictions"
-    lxc project create testovn -c features.networks=true -c features.images=false -c restricted=true
+    lxc project create testovn -c features.networks=true -c features.images=false -c features.storage.volumes=false -c restricted=true
```

Comment:
what is this part for?

Reply:
This was made in an attempt to figure out why `u1` was way slower to boot than any other instances. It's possibly not needed but it aligns with the other projects also having that same key/value set.